DiztinGUIsh/window/AliasList.cs

```diff
@@ -4,26 +4,24 @@
 using System.ComponentModel;
 using System.Globalization;
 using System.IO;
-using System.Linq;
-using System.Text.RegularExpressions;
 using System.Windows.Forms;
```

Comment:
rename this file to "LabelList" or similar.

better: make it based on the new MVC stuff.